### ISSUE TITLE
Blank page when accessing Instagram website

### ISSUE THREAD

**Issue by onmyouji** [brave/chromium#42]
I got blank page when trying to view Instagram website. Have tried to disable the Brave Shield and it's still not working (also tried to uninstall and install from Play Store).

Android version: 6.0
Brave 1.0.3

**Comment by srirambv** [comment]
Doesn't login to Instagram with shields down

Android 5.0.1/Yoga Tablet
Brave 1.0.3

**Comment by SergeyZhukovsky** [comment]
I cannot reproduce it on my devices and on x86 emulator also. @onmyouji could you tell me please what device do you have?

**Comment by onmyouji** [comment]
Motorola Moto X (2nd Gen)

**Comment by srirambv** [comment]
@SergeyZhukovsky  Reinstalling the new Brave from store allows me to login to Instagram

**Comment by SergeyZhukovsky** [comment]
@srirambv did you have the same version before? As I have no idea still what is wrong with that.

**Comment by srirambv** [comment]
@SergeyZhukovsky Yes its the same version. It logged in on the first attempt itself.

**Comment by onmyouji** [comment]
@SergeyZhukovsky I've found the problem.

I set my settings on "Advanced - Site settings - JavaScript" to Blocked. 
I thought by doing this, I can block JavaScript on all websites by default, so that I can enable them on certain sites only (by turning off "Block Scripts" on the shield settings). Turns out it doesn't work.

Changing Advanced - Site settings - JavaScript to Allowed solve the problem. Sorry for the trouble.
